Lib/test/test_compile.py

```diff
@@ -1384,53 +1384,90 @@ def check_op_count(func, op, expected):
                     actual += 1
             self.assertEqual(actual, expected)
 
+        def check_num_consts(func, typ, expected):
+            num_consts = 0
+            consts = func.__code__.co_consts
+            for instr in dis.Bytecode(func):
+                if instr.opname == "LOAD_CONST" and isinstance(consts[instr.oparg], typ):
+                    num_consts += 1
+            self.assertEqual(num_consts, expected)
+
         def check_consts(func, typ, expected):
-            slice_consts = 0
+            all_consts = set()
             consts = func.__code__.co_consts
             for instr in dis.Bytecode(func):
                 if instr.opname == "LOAD_CONST" and isinstance(consts[instr.oparg], typ):
-                    slice_consts += 1
-            self.assertEqual(slice_consts, expected)
+                    all_consts.add(consts[instr.oparg])
+            self.assertEqual(all_consts, expected)
 
         def load():
             return x[a:b] + x [a:] + x[:b] + x[:]
 
+        check_op_count(load, "BINARY_SLICE", 3)
+        check_op_count(load, "BUILD_SLICE", 0)
+        check_consts(load, slice, {slice(None, None, None)})
+
         def store():
             x[a:b] = y
-            x [a:] = y
+            x[a:] = y
```

Comment:
This whitespace *might* be intentional... `load` has the exact same whitespace above. We are testing the compiler, after all.

I'm inclined to just leave it.